### PR TITLE
Ffx and windows fixes

### PR DESCRIPTION
**Pull request by huitema**
There call to `alloca()` in the ECB tests does not compile on Windows; writing an `#ifdef` escape. Also, we missed a memory leak in the ffx tests, which is an easy fix.

### CODE REVIEW

File: t/picotls.c [hunk 1]
```diff
@@ -187,7 +187,11 @@ static void test_ecb(ptls_cipher_algorithm_t *algo, const void *expected, size_t
                                   16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31},
                          plaintext[] = {0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77,
                                         0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff};
+#ifdef _WINDOWS
+    uint8_t *actual = _malloca(expected_len);
+#else 
     uint8_t *actual = alloca(expected_len);
+#endif
```
Comment (kazuho) [inline t/picotls.c]: How about doing `actual = malloc(...)`, `assert(actual != NULL)` and `free(actual)` on all platforms? IMO the less #if we have, the less chance of having errors on some platforms.

Comment (huitema) [reply t/picotls.c]: I thought that you needed alloc on the stack for performance reason, but yes a standard malloc will work fine too.

Comment (kazuho) [reply t/picotls.c]: Never mind about performance, it's the test code :+1:

Comment (huitema) [reply t/picotls.c]: Done.

**Pull request merged**